### ISSUE TITLE
evdev 1.1.0 fails to install on Travis-CI

### ISSUE THREAD

**Issue by ddemidov** [gvalkov/python-evdev#98]
The command `pip install evdev` fails on Travis-CI.

See
* https://github.com/ev3dev/ev3dev-lang-python/pull/500,
* https://travis-ci.org/ev3dev/ev3dev-lang-python/builds/423161128#L545

The error is
```
gcc -pthread -Wno-unused-result -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -I/opt/python/3.4.6/include/python3.4m -c evdev/uinput.c -o build/temp.linux-x86_64-3.4/evdev/uinput.o -std=c99 -Wno-error=declaration-after-statement
  evdev/uinput.c: In function ‘uinput_setup’:
  evdev/uinput.c:92:28: error: ‘FF_MAX_EFFECTS’ undeclared (first use in this function)
       uidev.ff_effects_max = FF_MAX_EFFECTS;
                              ^
  evdev/uinput.c:92:28: note: each undeclared identifier is reported only once for each function it appears in
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for evdev
```

**Comment by ddemidov** [comment]
The workaround we currently use is `pip install evdev==1.0.0`

**Comment by dlech** [comment]
https://github.com/torvalds/linux/commit/33b96d934902f96e901b72ac18bbc47afad1ac20

You need 4.4 or newer kernel headers.

python-evdev could add 

```c
#ifndef FF_MAX_EFFECTS
#define FF_MAX_EFFECTS	FF_GAIN
#endif
```

for backwards compatibility

**Comment by gvalkov** [comment]
Thanks for the catch!. Fixed in evdev 1.1.2.
